solutions/azure-hub-spoke/README.md

```diff
@@ -38,54 +38,60 @@ Create a resource group for the deployment.
 az group create --name hub-spoke --location eastus
 ```
 
+> The location for the deployed resources defaults to the location used for the target resource group. This deployment uses availability zones for all resources that support it, as hub networks are usually business critical. This means  if the resource group's location does not support availability zones, you must provide an additional parameter to your chosen command below of `location=value` with a value supports availability zones. See [Azure regions with availability zones](https://learn.microsoft.com/azure/availability-zones/az-overview#azure-regions-with-availability-zones).
+
 **Basic deployment**
 
 Run the following command to initiate the deployment. If you would like to also deploy this sample with virtual machines and / or an Azure VPN gateway, see the `az deployment group create` examples found later in this document.
 
 ```azurecli-interactive
 az deployment group create \
     --resource-group hub-spoke \
-    --template-uri https://raw.githubusercontent.com/mspnp/samples/master/solutions/azure-hub-spoke/azuredeploy.json
+    --template-uri https://raw.githubusercontent.com/mspnp/samples/main/solutions/azure-hub-spoke/azuredeploy.json
 ```
 
 **Deploy with virtual machines**
 
-Run the following command to initiate the deployment with a Linux VM deployed to the first spoke network.
+Run the following command to initiate the deployment with a Linux VM deployed to the first spoke network and a Windows VM deployed to the second spoke network.
 
 ```azurecli-interactive
 az deployment group create \
     --resource-group hub-spoke \
-    --template-uri https://raw.githubusercontent.com/mspnp/samples/master/solutions/azure-hub-spoke/azuredeploy.json \
-    --parameters adminPassword=Password2020! linuxVMCount=1 windowsVMCount=1
+    --template-uri https://raw.githubusercontent.com/mspnp/samples/main/solutions/azure-hub-spoke/azuredeploy.json \
+    --parameters deployVirtualMachines=true adminUsername=azureadmin adminPassword=Password2023!
```

Comment:
Kill the hardcoded passwords, and let user know it'll prompt for them.

Reply:
I was going to have this always prompt, but that breaks the "no VM" flows (damn shared bicep file).  So, I added a warning.  What do you think @hallihan?